releasenotes/notes/0.4/remove-ibm-sampler-7779165095fd2a5f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    ``IBMSampler`` class which was deprecated earlier has been removed.
+    Use :class:`~qiskit_ibm_runtime.Sampler` class going forward.
+
```

Comment:
@dtquai One last change, could you please move this file under the `releasenotes/notes` folder. It should not be in 0.4 folder. When I do the release I will move it to 0.5 folder.